src/apis/tokenStorage.ts

```diff
@@ -0,0 +1,13 @@
+import storageFactory from '@/utils/storageFactory';
+
+export const getTokenStorage = () => {
+  const { getItem: localGetItem } = storageFactory(localStorage);
+  const { getItem: sessionGetItem } = storageFactory(sessionStorage);
+  if (localGetItem('MyToken', '')) {
+    return localStorage;
+  } else if (sessionGetItem('MyToken', '')) {
+    return sessionStorage;
+  } else {
+    return localStorage; // 기본값으로 localStorage 설정
+  }
```

Comment:
```suggestion
  if (sessionGetItem('MyToken', '')) {
    return sessionStorage;
  } 
  return localStorage; // 기본값으로 localStorage 설정
```
이렇게 줄일 수 있을 것 같습니다!! 👍

Reply:
좋은 의견 감사합니다!! 반영해서 커밋올리겠습니다 ㅎㅎ